### ISSUE TITLE
Link in `SetRadiusBlipEdge`'s description doesn't work anymore

### ISSUE THREAD

**Issue by JajarGaming** [citizenfx/natives#900]
The image preview link doesn't work anymore.

https://github.com/citizenfx/natives/edit/master/HUD/SetRadiusBlipEdge.md

**Comment by Z3rio** [comment]
When testing this native, it seems to be doing absolutely no difference to radar blips, nor normal blips.
Do you know what it is supposed to do? If so, I would love to fix the image :)

**Comment by JajarGaming** [comment]
No clue lol

**Comment by Z3rio** [comment]
@blattersturm Do you have any clue as to how `SetRadiusBlipEdge` works?
So I could go ahead and fix the image link.
(read convo above for more context)

**Comment by JajarGaming** [comment]
> @blattersturm Do you have any clue as to how `SetRadiusBlipEdge` works?
> 
> So I could go ahead and fix the image link.
> 
> (read convo above for more context)

Don't randomly ping them bruh

**Comment by Z3rio** [comment]
> > @blattersturm Do you have any clue as to how `SetRadiusBlipEdge` works?
> > So I could go ahead and fix the image link.
> > (read convo above for more context)
> 
> Don't randomly ping them bruh

It is not a "random ping", I would say it's quite fair to ping someone knowledgeable about the issue if no one has the answer.
I'm quite sure there are bigger issues than getting pinged once for a fair reason 😅.

**Comment by 4mmonium** [comment]
This is what it does:

![image](https://github.com/citizenfx/natives/assets/38232208/b7b8ab0d-ded2-4491-b596-b1bd8b42b43b)

And here's the code for it:

```lua
local coords = GetEntityCoords(PlayerPedId())
local radiusBlip = AddBlipForRadius(coords, 100.0) -- need to have .0
SetBlipColour(radiusBlip, 1)
SetBlipAlpha(radiusBlip, 255)
SetRadiusBlipEdge(radiusBlip, true)
```

I will fix the docs for that native soon.

**Comment by 4mmonium** [comment]
Resolved in #926 
Thank you for sending the report.
